fenix/app/src/androidTest/java/org/mozilla/fenix/ui/SearchTest.kt

```diff
@@ -83,13 +87,37 @@ class SearchTest {
     }
 
     @Test
-    fun searchScreenItemsTest() {
+    fun searchBarItemsTest() {
+        navigationToolbar {
+            verifyDefaultSearchEngine("Google")
+            verifySearchBarPlaceholder("Search or enter address")
+        }.clickUrlbar {
+            verifyKeyboardVisibility(true)
+            verifyScanButtonVisibility(true)
+            verifyVoiceSearchButtonVisibility(true)
```

Comment:
I told Andi, I think it's good practice to use named arguments in tests for readability.
Mind applying that throughout here? e.g, 
`verifyScanButtonVisibility(visible = true)`